portfolio/templates/index.html

```diff
@@ -1,6 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-JCX3Z8JZJC"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        gtag('config', 'G-JCX3Z8JZJC');
+    </script>
```

Comment:
I don't think we need this if we're providing it to sphinx

Reply:
ah yeah makes sense - I'll remove it and merge then